tests/foreman/api/test_docker.py

```diff
@@ -1083,146 +1083,77 @@ def setUpClass(cls):
             url=EXTERNAL_DOCKER_URL,
         ).create()
 
-    def setUp(self):
-        """Generate default ``DockerHubContainer`` keyword arguments.
-
-        Generate a dict of arguments that can be used when instantiating a
-        ``DockerHubContainer`` and save it as ``self.dhc_kwargs``.
-
-        """
-        self.container_name = gen_string('alphanumeric')
-        self.dhc_kwargs = {
```

Comment:
Name had to be removed, as we can't use the same name for 2 subtests. Without name this dict doesn't have much sense, as you'll write 1 line instead of 2 in tests, but have to store this setup section. Also there're tests which override `command` too, so it even introduces additional complexity sometimes.